### DISCUSSION TITLE
notifications

### DISCUSSION THREAD

**Discussion by 1746625968** [AvaloniaUI/Avalonia#9332] [Q&A]
![image](https://user-images.githubusercontent.com/50786228/108585020-cd29aa00-7380-11eb-852a-21e3256d042b.png)

I would like to ask how the notice shows the confirmation time, Yes and No, and how can I use it quickly

**Comment by pr8x** [comment]
Please ask questions in the discussions (https://github.com/AvaloniaUI/Avalonia/discussions).  Also I am not able to understand the question. Please try to reformulate it.

**Comment by maxkatz6** [comment]
ControlCatalog includes source for this custom notification

**Comment by timunie** [comment]
Converted this into Q&A as this is neither a bug nor a feature request. If you disagree please let us know.
